@here/harp-materials/lib/MapMeshMaterials.ts

```diff
@@ -385,6 +394,7 @@ export namespace FadingFeature {
         object: THREE.Object3D,
         fadeNear: number | undefined,
         fadeFar: number | undefined,
+        maxVisibility: number | undefined,
```

Comment:
it seems that usages of this are always defined, which means that the cameraToWorldDistance could just take the maxVisibility and doesn't need to take the camera... or do I miss something?

Reply:
No, you did not miss anything, although I left a possibility to calculate fading in old fashion (way). You simply do not pass maxVisibility - then camera far plane applies.

Reply:
Should I remove this option?

Reply:
Yeah, if you don't need a particular code path, best remove, we can always add it back later.